lib/packager.js

```diff
@@ -23,14 +23,14 @@ module.exports = {
     var buf = fs.readFileSync(path.resolve(__dirname, './client.js'));

```

Comment:
could we replace this file by using browserify in the future?

Reply:
good plan, I know there were issues with browserify a long time ago, which is why we opted for a custom packager, but it may be that things have moved on enough for us to do this, I have created an issue: https://github.com/happner/happn-3/issues/235